package.json

```diff
@@ -95,6 +95,7 @@
     "superagent": "1.2.0",
     "tinymce": "4.2.8",
     "to-title-case": "0.1.5",
+    "tv4": "1.2.7",
```

Comment:
While the package is released under public domain, meaning no attribution should be required ([IANAL](https://en.wikipedia.org/wiki/IANAL)), we might consider including this in `CREDITS.md`.